onnxruntime/core/providers/cuda/cuda_provider_factory.cc

```diff
@@ -15,10 +15,12 @@ namespace onnxruntime {
 struct CUDAProviderFactory : IExecutionProviderFactory {
   CUDAProviderFactory(OrtDevice::DeviceId device_id,
                       size_t cuda_mem_limit = std::numeric_limits<size_t>::max(),
-                      ArenaExtendStrategy arena_extend_strategy = ArenaExtendStrategy::kNextPowerOfTwo) 
+                      ArenaExtendStrategy arena_extend_strategy = ArenaExtendStrategy::kNextPowerOfTwo,
+                      int cudnn_conv_algo_search = 1) 
```

Comment:
> = 1 [](start = 48, length = 4)

It seems that default is 0 at other initialization, is there any reason why it's 1 here?